cmd/strapi-demo/createdemodata/start.go

```diff
@@ -0,0 +1,212 @@
+	User user   `json:"user"`
+}
+type user struct {
+	ID       int    `json:"id"`
+	Username string `json:"username"`
+	Email    string `json:"email"`
+	IsAdmin  bool   `json:"isAdmin"`
+}
+
+// GetStartCmd returns the Cobra start command.
+func GetStartCmd() *cobra.Command {
+	startCmd := createStartCmd()
+
+	createFlags(startCmd)
+
+	return startCmd
+}
+func createStartCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "create-demo-data",
+		Short: "Create demo data",
+		Long:  "Start populating data in strapi with default studentcards and transcripts",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			hostURL, err := cmdutils.GetUserSetVar(cmd, adminURLFlagName, adminURLEnvKey)
+			if err != nil {
+				return err
+			}
+			parameters := &strapiDemoParameters{
+				client:   &http.Client{},
+				adminURL: hostURL,
+			}
+			return startStrapiDemo(parameters)
+		},
+	}
+}
+
+func createFlags(startCmd *cobra.Command) {
+	startCmd.Flags().StringP(adminURLFlagName, adminURLFlagShorthand, "", adminURLFlagUsage)
+}
+
+// For Demo you can verify the records by browsing http://localhost:1337/admin/
+func startStrapiDemo(parameters *strapiDemoParameters) error {
+	if parameters.adminURL == "" {
+		return errMissingAdminURL
+	}
+
+	var client = parameters.client
+
+	adminUserValues := map[string]string{
+		"username": "strapi",
+		"email":    "user@strapi.io",
+		"password": "strapi"}
+
+	authToken, err := createAdminUser(client, parameters.adminURL, adminUserValues)
+	if err != nil {
+		return err
+	}
+	// dummy data for demo purposes
+	studentRecord1 := map[string]string{
+		"studentid":  "1234568",
+		"name":       "Tanu",
+		"university": "Faber College",
+		"semester":   "3",
+		"issuedate":  "2019-01-02T00:00:00.000Z",
+	}
+	studentRecord2 := map[string]string{
+		"studentid":  "323456898",
+		"name":       "Derek",
+		"university": "Faber College",
+		"semester":   "2",
+		"issuedate":  "2019-03-02T00:00:00.000Z",
+	}
+	transcriptRecord1 := map[string]string{
+		"studentid":    "323456898",
+		"name":         "Tanu",
+		"university":   "Faber College",
+		"status":       "graduated",
+		"totalcredits": "100",
+		"course":       "Bachelors'in Computing Science",
+	}
+	transcriptRecord2 := map[string]string{
+		"studentid":    "1234568",
+		"name":         "Derek",
+		"university":   "Faber College",
+		"status":       "graduated",
+		"totalcredits": "200",
+		"course":       "Bachelors'in Computing Science",
+	}
+
+	createOrFetchRecord(client, authToken, parameters.adminURL+adminURLEndpoint, "POST", studentRecord1)
+	createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint, "POST", studentRecord2)
+	createOrFetchRecord(client, authToken, parameters.adminURL+transcriptEndpoint, "POST", transcriptRecord1)
+	createOrFetchRecord(client, authToken, parameters.adminURL+transcriptEndpoint, "POST", transcriptRecord2)
+	createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint, "GET", nil)
```

Comment:
This get doesn't verify what was inserted, it merely lists all records. Can you add fetch by some id and verify that it equals inserted record?